.drone.yml

```diff
@@ -6408,6 +6408,7 @@ steps:
       - name: awsconfig
         path: /root/.aws
 
+  # Uploads to Houston
```

Comment:
I thought we had sunset Houston?

Reply:
We are in the process of getting rid of it. For now it (appears) that we're still using it. Left a comment here to help whoever comes along next that is under the assumption that Houston is no longer used anywhere.